lib/hanami/router.rb

```diff
@@ -86,7 +88,7 @@ def call(env)
 
       unless endpoint
         return not_allowed(env) ||
-               not_found
+               @default_app.call(env)
```

Comment:
@katafrakt Please don't change the level of abstraction: `not_found` represents the "not found" high level concept, whereas `@default_app` is an implementation detail. This line should be reverted.